packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,6 +30,14 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
+  const getDelayedPromise = () => {
```

Comment:
@prateekbh not that I know of, I just meant it probably made sense to wait until my changes were merged before spending time on this, since I changed a lot of stuff. But that's happened now.